dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/blas/MatrixFactory.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2015 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.dolphin.dnn.blas;
+
+/**
+ * Factory interface for {@link Matrix}.
+ */
+public interface MatrixFactory {
+
+  /**
+   * Creates a row vector.
+   * @param length the length of a row vector
+   * @return a generated row vector
```

Comment:
For the matrix implementation based on 'jblas', the returned vector is initialized with zeros by [Java spec](http://docs.oracle.com/javase/specs/jls/se7/html/jls-4.html#jls-4.12.5). So, `create()` and `zeros()` make the same results. However, it is possible that the returned vector may not be initialized depending on the backend libraries.
Do we should change the comment to "Creates an uninitialized row vector"?